Storage/Storage/Model/ShippingLine+CoreDataProperties.swift

```diff
@@ -14,5 +14,23 @@ extension ShippingLine {
     @NSManaged public var total: String?
     @NSManaged public var totalTax: String?
     @NSManaged public var order: Order?
+    @NSManaged public var taxes: Set<ShippingLineTax>?
```

Comment:
just double checking, the taxes don't have to be ordered right?

Reply:
Not at the moment, this is not displayed anywhere but we need them to know what tax id to refund!